bcda/bcdacli/cli_test.go

```diff
@@ -836,3 +834,110 @@ func getRandomPort(t *testing.T) int {
 
 	return listener.Addr().(*net.TCPAddr).Port
 }
+
+func CreateTestZipFile(file string) error {
+	zf, err := os.Create(file)
+	if err != nil {
+		return err
+	}
+	defer zf.Close()
+
+	w := zip.NewWriter(zf)
+	var files = []string{
+		"T.BCD.A0001.ZC8Y18.D181120.T1000001",
+		"T.BCD.A0001.ZC8Y18.D181120.T1000002",
+		"T.BCD.A0001.ZC8Y18.D181120.T1000003",
+	}
+
+	for _, file := range files {
+		f, err := w.Create(file)
+		if err != nil {
+			return err
+		}
+		_, err = f.Write([]byte("foo bar"))
+		if err != nil {
+			return err
+		}
+	}
+
+	return w.Close()
+}
+
+func getFileCount(path string) int {
+	f, _ := ioutil.ReadDir(path)
```

Comment:
We shouldn't ignore this error. To keep the return value clean, you can pass in a test object and use the `assert.NoError(t, err)` here. That way you can still return the file count while not ignoring the error.